docs/content/Deployment/Platform-Kubernetes.mdx

```diff
@@ -348,6 +351,8 @@ spec:
                   fieldPath: metadata.name
             - name: CUBESTORE_META_ADDR
               value: cubestore-router:9999
+            - name: CUBESTORE_DATA_DIR
+              value: /cube/.cubestore/worker/data
```

Comment:
@adnanrahic Is there a way to get the worker number and use it in the path here?

Reply:
I think you can use `$(POD_NAME)`.
Check this out.
https://github.com/cube-js/cube.js/blob/a345c850868334be0d404cf6476f7d42a1fad2b0/examples/kubernetes/cluster/cubestore-workers-statefulset.yaml#L33

Can you see how I use `cubestore-workers-0...` for the `CUBESTORE_WORKERS` env var here?
https://github.com/cube-js/cube.js/blob/a345c850868334be0d404cf6476f7d42a1fad2b0/examples/kubernetes/cluster/cubestore-workers-statefulset.yaml#L39

The `$(POD_NAME)` will be `cubestore-workers-0` for the first worker, `cubestore-workers-1` and `cubestore-workers-2` respectively for the rest. Here's why.
https://github.com/cube-js/cube.js/blob/a345c850868334be0d404cf6476f7d42a1fad2b0/examples/kubernetes/cluster/cubestore-workers-statefulset.yaml#L20-L23

Let me know if you need more help, we can work together on figuring it out.

Reply:
@YakovlevCoded left me a `<GitHubCodeBlock />` component, so I can directly reference these files instead of duplicating them in the docs 💯